### PR TITLE
rename protocols page

### PR DESCRIPTION
**Pull request by pwright**
@clebertsuconic  as discussed, but I'm not sure if this makes sense given the content of the page, alternative might be 'Supported Protocols and APIs'?

### CODE REVIEW

File: docs/user-manual/en/SUMMARY.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 * [Using the Server](using-server.md)
 * [Upgrading](upgrading.md)
 * [Address Model](address-model.md)
-* [Protocols and Interoperability](protocols-interoperability.md)
```
Comment (clebertsuconic) [inline docs/user-manual/en/SUMMARY.md:13]: but you did not rename the md here.. did you?

Comment (gtully) [reply docs/user-manual/en/SUMMARY.md:13]: a quick search, I see protocols-interoperability used in links in mqtt.md amqp.md etc

Comment (pwright) [reply docs/user-manual/en/SUMMARY.md:13]: I updated the PR, however I think these links show that the chapter *should not* be renamed.

ie "interoperability"  helps reader understand the scope of the doc, it's not just a list of supported protocols.

Why would i click on 'Supported protocols' to read about 'Configuration of max frame payload length for STOMP web-socket'?

**Comment by pwright** [comment]
closing as 'not a good change'

**Pull request closed without merging**